Templates/StreamDeck.PluginTemplate.Csharp/content/RegisterPluginAndStartStreamDeck.sh

```diff
@@ -16,19 +14,25 @@ uuid=$(sed -n 's/.*"UUID": "\(.*\)"/\1/p' manifest.json)
 
 pluginName=$(echo "${uuid}" | tr -d \" | sed -e "s/${SUFFIX}$//")
 
+#Define Plugin Directory
+PLUGIN_DIR="${HOME}/Library/Application Support/com.elgato.StreamDeck/Plugins/${pluginName}.sdPlugin"
+
 #Notify user of Action.
 echo "Installing the ${pluginName} plugin..."
 
 #On first run the file does not exist, we check if it exists.
-if [ -d ${PLUGIN_DIR} ]; then rm -fr ${PLUGIN_DIR}; fi
+[[ -d "${PLUGIN_DIR}" ]] && rm -fr "${PLUGIN_DIR}"
```

Comment:
AH! That might be why your PR will work and the way we'd been doing it kept failing.